UnityProject/Assets/Scripts/Chemistry/ReagentMix.cs

```diff
@@ -215,6 +271,11 @@ public float Total
 			get { return reagents.Sum(kvp => kvp.Value); }
 		}
 
+		public bool Contains(Reagent reagent)
+		{
+			return reagents.ContainsKey(reagent);
+		}
+
```

Comment:
Is this needed? you shouldn't need to know about the keys and you can use `reagentMix[reagent]` instead.